javascript-if/main.js

```diff
@@ -0,0 +1,61 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(num) {
+  return num < 5;
+}
+
+function isEven(num) {
+  return (num % 2 === 0);
+}
+
+function startsWithJ(string) {
+  return string[0] === 'J';
+}
+
+function isOldEnoughToDrink(person) {
+  return person.age >= 21;
+}
+function isOldEnoughToDrive(person) {
+  return person.age >= 16;
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return person.age < 0;
+}
```

Comment:
Unborn children are allowed to? Wish I knew that.

Reply:
**only** unborn children allowed
...had to get that false return somehow lol